### ISSUE TITLE
SEO identify how we can update metadata and include schema on record details pages to improve and enhance search results

### ISSUE THREAD

**Issue by grange85** [nationalarchives/findability#7]
- [ ] Confirm what series we need to apply this to (WO 372, PROB 11)
- [ ] Identify what information can be included in the schema 
- [ ] Identify how the basic page metadata (title, description, keywords etc.) can be improved to dynamically reflect the content of the details page

## Useful links
- [schema.org](https://schema.org/CreativeWork}
- [JSON-LD](https://en.wikipedia.org/wiki/JSON-LD)

**Comment by grange85** [comment]
This is how a Discovery WO 372 record looks like in Google
![image](https://user-images.githubusercontent.com/783875/44463496-d13dab00-a60f-11e8-941d-8268a5002313.png)

[Medal card of Jensen, Stanley](http://discovery.nationalarchives.gov.uk/details/r/D3089484)

**Comment by grange85** [comment]
This is how a Discovery PROB 11 record looks in Google
![image](https://user-images.githubusercontent.com/783875/44467196-25e62380-a61a-11e8-91b9-ee404b0058f8.png)
[Will of George Stanley of Paultons, Hampshire](http://discovery.nationalarchives.gov.uk/details/r/D694274)

**Comment by grange85** [comment]
I just did a Google [search for _Stanley Jensen medals_](https://www.google.com/search?safe=active&client=firefox-b-ab&bih=943&biw=1920&ei=F2p9W4n3A4n5wQKKsL2YBQ&q=stanley+jensen+medals&oq=stanley+jensen+medals&gs_l=psy-ab.3...0.0.0.2899.0.0.0.0.0.0.0.0..0.0....0...1c..64.psy-ab..0.0.0....0.0DbmEeWsO4U) and our record was nowhere to be found

**Comment by cariannewhitworth** [comment]
Thanks for this @grange85.

@netdesignr could you tell us where the Google title is pulling its info from for each of these different types of pages (is it the record title) and where the description is pulling from? It looks like half of it is something, and the second half is pulling from the record description text?

**Comment by cariannewhitworth** [comment]
THOUGHTS FROM CEE EDITORIAL ON SEARCH

- Both Chrissy and Matt agreed that the two series Matt H isolated for testing would be a good starting point. Chrissy also raised for some searches (ie topics, not names) the topic itself isn't mentioned in the record description (where we talked about grabbing the data from) but is actually a level or two above in the series. Made me wonder whether we'd need different meta data programming for different parts of the catalogue - but as I don't know the SEO problems well enough yet this may not be a prob at all.

- Matt and Chrissy raised an interesting point around how a BIG problem the contact centre has to deal with is telling users what they DON'T have... because our catalogue entries sometimes rank in Google above the archives that actually have the records. Task here around asking them to give us a list of the biggest problems? They said they know it so well they could reel it off by memory... interesting

**Comment by grange85** [comment]
Would it be worth peeling this off into a number of clear and separate tasks?

E.g.

1. adding schema.org/JSON-LD to details pages in WO 372 and PROB 11
2. improving embedded metadata (titles, descriptions, keywords)
3. consider the layout of the visible page to improve both SEO and user experience 

A lot of the above is either not relevant for the catalogue (image optimization and outbound links) or way beyond the scope of the project (anything that involves changing catalogue descrtiptions)

While the checklist is valuable - I'm concerned that it's distracting from the more a immediate and achievable tasks (probably 1 and 2 (and maybe 3) above)

@netdesignr @cariannewhitworth @domingobishop

**Comment by cariannewhitworth** [comment]
Absolutely, thanks Andy. 

On @domingobishop's point from yesterday, how do we want to measure improvement on this?

@netdesignr can we get 1 and 2 done by next Fri, on the 31 Aug? And can we also track what that takes (time/effort wise) so we can use that to size the work it'll take to do other series?

**Comment by netdesignr** [comment]
That's right @grange85, as I've mentioned before, this checklist won't reflect a particular section on the website. We will apply things from it whenever it's needed, depending on the project.

@cariannewhitworth there are a couple of ways to measure improvement on this, but first, we need to understand what's wrong with it, identify the meta, define how we want that page to be indexed by Google and then set the SEO expectations.

I'm not entirely sure what's the definition of Done in this scenario. To define/identify the right meta for both page and schema, keyword research, build the schema, implement everything in Discovery on a specific subpage and deploy that on a test environment, I think it can take more than a week, but it depends on how many people get involved. For example, I'm not in charge with test/live deployments etc..

**Comment by netdesignr** [comment]
Yes @grange85, I personally think that's better to keep this as a reference issue.

**Comment by cariannewhitworth** [comment]
Thanks both, this is ace. A few thoughts on keeping the scope of this manageable, as a follow up to what Chrissy and Matt said yesterday.

Because of the variance across different catalogue descriptions, it sounds like 'successful' metadata tagging would essentially mean a degree of custom tagging for different series, instead of a catch-all 'just pull in the title or catalogue description' instruction. Because the custom approach could all too easily evolve into a complex project (I can just imagine the stakeholder debates about how to determine the seo 'value' for each different series now!) I wonder whether we do this: 

- Take metrics on the two series Matt H selected, on how they're performing now. 
- Have a quick check on how other archives, like NARA, are SEOing their content in Google in case it can guide us. 
- Make an executive decision: remove that repetitive line about 'uk gov archives', insert date and/or catalogue description, and implement it. (Take a note of the work it took to do this, for this series size, so we can use to size remaining seo work).
- Leave it for a while and move on to our other objectives. Check in to see how it's performing later and make decisions about how to adjust it based on what we've learned.

(Left out the sprint plan/dev list as Mihai and Andy you've covered this above) but the thinking is take our best guess, deliver a bit of it asap and evaluate, if that seems like a reasonable approach?
